src/search/search_engines/search_common.h

```diff
@@ -68,15 +68,13 @@ extern std::shared_ptr<OpenListFactory> create_wastar_open_list_factory(
     const options::Options &opts);
 
 /*
-  Create open list factory and f_evaluator (used for displaying progress
-  statistics) for A* search.
+  Create open list factory for A* search.
 
   The resulting open list factory produces a tie-breaking open list
-  ordered primarily on g + h and secondarily on h. Uses "eval" from
-  the passed-in Options object as the h evaluator.
+  ordered primarily on g + h and secondarily on h.
```

Comment:
This is the part where I don't like the way the new function is split up: the comment says that the open list is ordered by g+ h, but the parameters of the functions are h and f, so the comment is only true if the user puts in the correct f.